src/sentry/api/endpoints/project_codeowners.py

```diff
@@ -39,9 +39,12 @@ def validate_association(
         sentry_items = [item.external_name for item in associations]
 
     diff = [str(item) for item in raw_items if item not in sentry_items]
```

Comment:
I think we can just make this a set 
```python
diff = {str(item) for item in raw_items if item not in sentry_items}
```

Reply:
@MeredithAnya yes, but this does not keep the order of insertion, which causes the tests to become flakey.

Reply:
@NisanthanNanthakumar if the order doesn't matter except for the tests, I would probably go with just updating the tests for this. If we do want to keep the usernames/emails in some specific order then would probably keep what you have, but it just seemed to me that that wasn't necessary.